sample_size/sample_size_calculator.py

```diff
@@ -10,6 +12,91 @@
 
 DEFAULT_ALPHA = 0.05
 DEFAULT_POWER = 0.8
+DEFAULT_VARIANTS = 2
+METRICS_SCHEMA = {
```

Comment:
should we put this in a `schemas/metrics_schema.json` and load it instead of putting here?

Reply:
Do we have to make a schema folder though there's only one schema in total? Can I put the file with other script like the latest commit does?